src/main/java/edu/harvard/iq/dataverse/util/json/JsonParser.java

```diff
@@ -566,7 +553,8 @@ public DatasetField parseField(JsonObject json, Boolean testType) throws JsonPar
     
 
         if (type == null) {
-            throw new JsonParseException("Can't find type '" + json.getString("typeName", "") + "'");
+            logger.info("Can't find type '" + json.getString("typeName", "") + "'");
```

Comment:
Is this going to happen a lot? If so, should it be reduced from "info" to "fine" so it doesn't clutter the logs?

Reply:
@pdurbin This'll happen potentially multiple thousands of times when doing a harvesting run on a big OAI set for the first time. So I'll just change it to "fine".